### ISSUE TITLE
hpc-intro reference

### ISSUE THREAD

**Issue by tkphd** [hpc-carpentry/hpc-intro#136]
@psteinb, please create a Zenodo entry (and DOI) for hpc-intro. This would allow us to populate `CITATION` in harmony with the [Template](https://github.com/carpentries/workshop-template/blob/gh-pages/CITATION).

**Comment by psteinb** [comment]
Superb idea. I’ll do that ASAP tomorrow.

**Comment by psteinb** [comment]
Done, the webhook is installed. As documented [here](https://guides.github.com/activities/citable-code/), we should continue and make a release. Only through publishing a release, we'll get a DOI.

**Comment by reid-a** [comment]
Is this release process documented somewhere other than this issue? I'm concerned knowledge of the Zenodo/Github link will disappear if the issue is closed.
Also, I was not able to view the "webhooks' section of the settings, to verify that the link had survived migration, likely due to insufficient privilege.

**Comment by tkphd** [comment]
The webhook did indeed survive. Do we need a MAINTAINING.md to document behind-the-scenes stuff?

**Comment by reid-a** [comment]
Yes please, especially since there are apparently components that are invisible to non-administrator.
